oss-licenses-plugin/build.gradle

```diff
@@ -4,13 +4,13 @@ apply plugin: 'java'
 dependencies {
     implementation gradleApi()
     implementation localGroovy()
-    implementation 'com.android.tools.build:gradle:3.4.2'
+    implementation 'com.android.tools.build:gradle:3.5.1'
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.mockito:mockito-core:2.23.4'
 }
 
 group = 'com.google.android.gms'
-version = '0.10.0'
+version = '0.11.0'
```

Comment:
Shouldn't this be a bug fix since it doesn't fundamentally change functionality?

Reply:
Yea bug fix seems more right-er.